### ISSUE TITLE
Moves.json variables

### ISSUE THREAD

**Issue by homu-gamepress** [kwsch/ReMasters#2]
Hope this is a good place to drop notes, since the discord is too busy.

in moves.json:

u2: 4 = Unity Attacks
u4: [Move User] 0 = Pokemon; 1 = Trainer
u5: [Move Categories] 2 = Regular Moves; 3 = Sync Moves; 4 = Unity Moves
u7: [Move Target] 0 = An Ally; 1 = All Allies; 2 = An Opponent; 3 = All Opponents; 4 = Self; 7 = Allied Field of Play; 8 = Opponent Field of Play; 9 = Entire Field
u12: [Move Uses]

Not quite familiar with protobuf, so not sure I feel comfortable enough to make the code changes yet.

**Comment by kwsch** [comment]
Any changes to the repo for this would be done in [`Move.proto`](https://github.com/kwsch/ReMasters/blob/04413c81021846dc258c8e61183fc31ef0eec455/ReMastersLib/Proto/Move.proto), as the repo has been set up to instantly generate the c# handling for reading the pb file when it's changed.

Should be a pretty simple change; just define a new enum at the bottom of the file like the others, and change the type/name in the move proto definition :)
```proto
enum User {
    Pokemon = 0;
    Trainer = 1;
}

enum Group {
    None = 0;
    Unknown1 = 1;
    Regular = 2;
    Sync = 3;
    Unity = 4;
}

enum Target {
    AllySingle = 0;
    AllyAll = 1;
    OpponentSingle = 2;
    OpponentAll = 3;
    Self = 4;
    AllyField = 7;
    OpponentField = 8;
    EntireField = 9;
}
```

**Comment by homu-gamepress** [comment]
Alright, I been digging through a few other protos and will give that a whirl!

**Comment by kwsch** [comment]
Closed via #3
